### PR TITLE
[Doc-enhancement][Code snippets][2189318]

### PR DESCRIPTION
**Pull request by v-silakshmi**

### CODE REVIEW

File: msteams-platform/bots/how-to/conversations/channel-messages-with-rsc.md [hunk 1]
```diff
@@ -82,6 +82,31 @@ To sideload in a team to test, whether all channel messages in a team with RSC a
 |-------------|-------------|------|----|
 |Channel messages with RSC permissions|	Microsoft Teams sample app demonstrating on how a bot can receive all channel messages with RSC without being @mentioned.|	[View](https://github.com/OfficeDev/Microsoft-Teams-Samples/tree/main/samples/bot-receive-channel-messages-withRSC/csharp) |	[View](https://github.com/OfficeDev/Microsoft-Teams-Samples/tree/main/samples/bot-receive-channel-messages-withRSC/nodejs) |
 
+
+### Code snippets for RSC permissions
```
[the remark this answers is not in the document]
Comment (v-silakshmi) [reply msteams-platform/bots/how-to/conversations/channel-messages-with-rsc.md:86]: Made changes.

**Review by v-habibaumme — DISMISSED** [review]
Approved

**Review by v-habibaumme — APPROVED** [review]
Approved

**Review by Wajeed-msft — APPROVED** [review]

**Review by v-habibaumme — APPROVED** [review]
Approved

**Review by surbhigupta12 — APPROVED** [review]
Approved

**Pull request merged**